From fc2e4ba0020e533a64149d0b17d9e4f158704e07 Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Wed, 7 Jan 2026 12:36:15 +0100
Subject: [PATCH 04/26] Brad Barden (@IAMB) wrote bindings for Perl

---
 THANKS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/THANKS b/THANKS
index b962a786..9fd52e57 100644
--- a/THANKS
+++ b/THANKS
@@ -32,6 +32,7 @@ Alexander Morris (@alexpmorris)
 Amit Murthy (@amitmurthy)
 Andrew Bennett (@potatosalad)
 Andrew Lambert (@charonn0)
+Brad Barden (@IAMB)
 Bruce Mitchener (@waywardmonkeys)
 Bruno Oliveira (@abstractj)
 Caolan McMahon (@caolan)
-- 
2.49.0

