From 1a572b5028d46fe71afa525d71247c9d05eb832e Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Thu, 8 Jan 2026 12:00:32 +0100
Subject: [PATCH 10/26] Update ChangeLog

---
 ChangeLog | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ChangeLog b/ChangeLog
index 8a2c57e1..3a9d21af 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,7 @@
 * Version 1.0.21-stable
  - Export missing crypto_ipcrypt_nd_keygen() helper function.
  - Fixed compilation with GCC on aarch64.
+ - Libsodium can be directly used as a dependency in a Zig project.
 
 * Version 1.0.21
   This point release includes all the changes from 1.0.20-stable, which
-- 
2.49.0

