From 9a310ef9a4dd1980f933304dccaa288bfb9e137c Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Sun, 11 Jan 2026 20:03:13 +0100
Subject: [PATCH 13/26] Update ChangeLog

---
 ChangeLog | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 3a9d21af..6ef81ea1 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,4 +1,6 @@
 * Version 1.0.21-stable
+ - Performance: SHA3 (Keccak1600) now leverages ARM SHA3 instructions when
+available on ARM platforms.
  - Export missing crypto_ipcrypt_nd_keygen() helper function.
  - Fixed compilation with GCC on aarch64.
  - Libsodium can be directly used as a dependency in a Zig project.
-- 
2.49.0

