From 414c40318ae6e39a6cd39a968dfc4e4e26a8e54f Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Wed, 4 Feb 2026 23:07:01 +0100
Subject: [PATCH 22/26] Update ChangeLog

---
 ChangeLog | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index 741e7d76..8fa5ec0d 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -7,8 +7,12 @@ slower than without it.
 JavaScript, not just WebAssembly.
  - Performance: SHA3 (Keccak1600) now leverages ARM SHA3 instructions when
 available on ARM platforms.
+ - Performance: WebAssembly SIMD implementations of Argon2 have been added.
+ - Emscripten: a new option allows compilation with SIMD support.
  - Export missing crypto_ipcrypt_nd_keygen() helper function.
- - Fixed compilation with GCC on aarch64 and gcc 4.1
+ - Fixed compilation with GCC on aarch64 and gcc 4.1.
+ - Added dompatibility with Visual Studio 2026  when toolsets do not define
+PlatformToolsetVersion.
  - Libsodium can be directly used as a dependency in a Zig project.
 
 * Version 1.0.21
-- 
2.49.0

